### PR TITLE
Order card

### PR DESCRIPTION
**Pull request by rakeshrkamath**

### CODE REVIEW

File: src/components/common/orderCard/index.tsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+        <Label
+          content={cardDetails.Items + data.ItemText}
+          height="19px"
+          color="#4c4c4c"
+          fontFamily={`${fontFamilies.fontFamilyOsSemiBold}`}
+          fontSize="14px"
+          letterSpacing="0"
+          lineHeight="19px"
+          width="none"
+        />{" "}
```
Comment (naikashok) [inline src/components/common/orderCard/index.tsx]: Remove this {" "} in all places

File: src/components/common/orderCard/index.tsx [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+        <LineBreak />{" "}
+        <Label
+          content={data.costUnit + cardDetails.Costs}
+          height="19px"
+          color="#4c4c4c"
```
Comment (naikashok) [inline src/components/common/orderCard/index.tsx]: Again hardcoded???

File: src/components/common/orderCard/index.tsx [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+          fontFamily={`${fontFamilies.fontFamilyOsSemiBold}`}
+          fontSize="14px"
+          letterSpacing="0"
+          lineHeight="19px"
+          width="none"
+        />
+      </ItemsSelected>
+      <Delivery>Out for Delivery</Delivery>
```
Comment (naikashok) [inline src/components/common/orderCard/index.tsx]: Hardocoded

Comment (rakeshrkamath) [reply src/components/common/orderCard/index.tsx]: removed

File: src/components/common/orderCard/index.tsx (hunk 2 above)
Comment (rakeshrkamath) [reply src/components/common/orderCard/index.tsx]: removed

File: src/components/common/orderCard/index.tsx (hunk 1 above)
Comment (rakeshrkamath) [reply src/components/common/orderCard/index.tsx]: removed

**Review by naikashok — APPROVED** [review]

**Pull request merged**